types.go

```diff
@@ -1012,14 +1012,17 @@ type TKEY struct {
 	Mode       uint16
 	Error      uint16
 	KeySize    uint16
-	Key        string
+	Key        string `dns:"size-hex:KeySize"`
 	OtherLen   uint16
-	OtherData  string
+	OtherData  string `dns:"size-hex:OtherLen"`
 }
 
 func (rr *TKEY) String() string {
 	// It has no presentation format
```

Comment:
This comment is now not true anymore; we just made one up. What does dig show for TKEY?
As said if we make a format we should also be able to parse it back in *and* it needs to be interoperable with other implementations.

I don't think we should output anything here...

Reply:
I can live with what do for TSIG's String():
~~~
	s := "\n;; TSIG PSEUDOSECTION:\n"
	s += rr.Hdr.String() +
		" " + rr.Algorithm +
~~~
And make it TKEY PSEUDOSECTION or something

Now need to parse from text, don't do that for the (wireonly) TSIG either

Reply:
updated to be similar to TSIG - the parsing from text still somewhat worked so I left it in the newly added test as is right now.